shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -421,11 +441,25 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
       case kOpacity:
         embedded_view.alpha = (*iter)->GetAlphaFloat() * embedded_view.alpha;
         break;
-      case kBackdropFilter:
+      case kBackdropFilter: {
+        // We only support DlBlurImageFilter for BackdropFilter.
+        if ((*iter)->GetFilter().asBlur() && canApplyBlurBackdrop) {
+          // sigma_x is arbitrarily chosen as the radius value because Quartz sets
+          // sigma_x and sigma_y equal to each other. DlBlurImageFilter's Tile Mode
+          // is not supported in Quartz's gaussianBlur CAFilter, so it is not used
+          // to blur the PlatformView.
+          [blurRadii addObject:@((*iter)->GetFilter().asBlur()->sigma_x())];
+        }
         break;
+      }
     }
     ++iter;
   }
+
+  if (canApplyBlurBackdrop && ([blurRadii count] > 0)) {
```

Comment:
We can remove `canApplyBlurBackdrop`. `[blurRadii count] > 0` is only true when `canApplyBlurBackdrop` is true.
So just:
`if ([blurRadii count] > 0)`

Reply:
Whoops, you're right. I forgot that `canApplyBlurBackdrop` is included in the if() check on line 446.  Thanks, Chris!

Reply:
Offline conversation, reverting to original.